js/public/app.js

```diff
@@ -1473,8 +1476,11 @@ app.factory('Calendar', ['$rootScope', '$filter', 'VEventService', 'TimezoneServ
 			this._properties.order = order;
 			this._setUpdated('order');
 		},
-		get cruds() {
-			return this._properties.cruds;
+		get writable() {
+			return this._properties.writable;
+		},
+		get shareable() {
+			return this._properties.shareable;
```

Comment:
No, a CalDAV server can disallow sharing for certain calendars. 
https://trac.calendarserver.org/browser/CalendarServer/trunk/doc/Extensions/caldav-sharing.txt#L704

We are just getting rid of the can re share option, because it's giving a false sense of security.